HISTORY.md

```diff
@@ -18,6 +18,7 @@
 * Added new functions to write schema dump to a string rather than stdout, so that non-console interactive environments such as Jupyter will be able to capture and print the output. [#5026](https://github.com/TileDB-Inc/TileDB/pull/5026)
 * Improve dense read performance for small reads. [#5145](https://github.com/TileDB-Inc/TileDB/pull/5145)
 * Skip caching redirect uri on array create. [#5224](https://github.com/TileDB-Inc/TileDB/pull/5224)
+* Add serialization and API changes for post_array_schema_from_rest. [#5261](https://github.com/TileDB-Inc/TileDB/pull/5261)
```

Comment:
I would include "add `tiledb_array_schema_load_with_config` C API to load the schema of an array with configuration".

Reply:
I'm always debating including too much about serialization in the release notes when it's not a change that the end user can really use. This CAPI is just to give the REST server the ability to implement the route required so we can switch the API the user really uses in the following release. I don't really feel strongly here... @ihnorton any thoughts?

Reply:
The C API is likely useful outside REST; I am planning to expose it from C#.

Reply:
Ahhh I see, in that case I'll add your suggestion.